tests/test_ops.py

```diff
@@ -1734,9 +1790,241 @@ def test_mathtransform():
+        #{"opName": "bitcast", "outName": "bitcast/emptyArrayTest/from_uint32_to_float32", "varShapes": [[0]],"varTypes": ["uint32"], "varInit": ["empty"], "output": tf.float32},
+
+        #{"opName": "bitcast", "outName": "bitcast/from_float32_to_float64", "varShapes": [[2]], "varTypes": ["float32"], "varInit": ["uniform"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/from_bfloat16_to_float64", "varShapes": [[4]], "varTypes": ["bfloat16"],"varInit": ["uniform"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/from_half_to_float64", "varShapes": [[4]], "varTypes": ["half"], "varInit": ["uniform"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/from_float64_to_float64", "varShapes": [[1]], "varTypes": ["float64"], "varInit": ["uniform"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/from_int32_to_float64", "varShapes": [[2]], "varTypes": ["int32"], "varInit": ["uniform_int2"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/from_int64_to_float64", "varShapes": [[2]], "varTypes": ["int64"], "varInit": ["uniform_int2"], "output": tf.float64},
+        #{"opName": "bitcast", "outName": "bitcast/emptyArrayTest/from_uint32_to_float64", "varShapes": [[0]], "varTypes": ["uint32"], "varInit": ["empty"], "output": tf.float64},
+
+        # {"opName": "bitwise_or", "outName": "bitwise_or/rank2_int32", "varShapes":[[1,2], [1,2]], "varTypes":["int32", "int32"], "varInit":["uniform_int10", "uniform_int2"]},
+        # {"opName": "bitwise_or", "outName": "bitwise_or/rank2_int64", "varShapes": [[1, 2], [1, 2]],"varTypes": ["int64", "int64"], "varInit": ["uniform_int10", "uniform_int2"]},
+        # {"opName": "bitwise_or", "outName": "bitwise_or/rank3_int32", "varShapes":[[1,1, 2], [1,1, 2]], "varTypes":["int32", "int32"], "varInit":["uniform_int10", "uniform_int2"]},
+        # {"opName": "bitwise_or", "outName": "bitwise_or/rank3_int64", "varShapes": [[1, 1, 2], [1, 1, 2]],"varTypes": ["int64", "int64"], "varInit": ["uniform_int10", "uniform_int2"]},
+
+        #{"opName": "bitwise_or", "outName": "bitwise_or/emptyArrayTest/rank2_int32", "varShapes":[[0,0], [0,0]], "varTypes":["int32", "int32"], "varInit":["uniform_int10", "empty"]},
+        #{"opName": "bitwise_or", "outName": "bitwise_or/emptyArrayTest/rank2_int64", "varShapes": [[0,0], [0,0]],"varTypes": ["int64", "int64"], "varInit": ["uniform_int10", "empty"]},
+        #{"opName": "bitwise_or", "outName": "bitwise_or/emptyArrayTest/rank3_int32", "varShapes": [[0, 0, 0], [0, 0, 0]], "varTypes": ["int32", "int32"], "varInit": ["uniform_int10", "empty"]},
+        #{"opName": "bitwise_or", "outName": "bitwise_or/emptyArrayTest/rank3_int64", "varShapes": [[0, 0, 0], [0, 0, 0]], "varTypes": ["int64", "int64"], "varInit": ["uniform_int10", "empty"]},
```

Comment:
👍 
But switch some of these empty for empty shapes like [2, 0, 1] etc, not just the "all zeros" cases.
Same for bitwise xor and log softmax below.

Reply:
Fixed